test/e2e/cluster_status_test.go

```diff
@@ -30,6 +30,41 @@ import (
 	"k8s.io/client-go/pkg/api/v1"
 )
 
+func TestReadyPodsStatus(t *testing.T) {
+	f := framework.Global
+	size := 1
+	testEtcd, err := createCluster(t, f, newClusterSpec("test-etcd-", size))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	defer func() {
+		if err := deleteEtcdCluster(t, f, testEtcd); err != nil {
+			t.Fatal(err)
+		}
+	}()
+
+	if _, err := waitUntilSizeReached(t, f, testEtcd.Metadata.Name, size, 60*time.Second); err != nil {
+		t.Fatalf("failed to create %d members etcd cluster: %v", size, err)
+	}
+
+	err = retryutil.Retry(5*time.Second, 3, func() (done bool, err error) {
+		checkedEtcdCluster, err := k8sutil.GetClusterTPRObject(f.KubeClient.CoreV1().RESTClient(), f.Namespace, testEtcd.Metadata.Name)
+		if err != nil {
+			t.Logf("failed to get updated cluster object: %v", err)
+			return false, nil
+		}
+		if len(checkedEtcdCluster.Status.ReadyMembers) != size {
```

Comment:
`("size of ready members want = %d, get = %d (%v). Will retry checking ReadyMembers", size, len(m), m)`